sptx_format/README.md

```diff
@@ -71,15 +71,15 @@ The field of view is the most complex file in the spaceTx format, and must be cr
 It provides two key types of information: information about the field of view, and information about each tile contained in it.
 
 The field_of_view.json file specifies the shape of the image tensor, including the size of the (X, Y) image in pixels, and the number of z-planes, imaging channels, and imaging rounds in the experiment.
-Thus, an image tensor has shape (r, c, z, y, x).
+Thus, an image tensor has shape (r, c, z, y, x), though y and x are limited to at most 3000 pixels.
```

Comment:
@ambrosejcarr, @dganguli

Reply:
I support this limit. I haven't seen anyone hit this size, it will just prevent people from sending us stitched experiments that aren't chopped into FoVs.